src/assets/images/CloseIcon.jsx

```diff
@@ -1,4 +1,6 @@
-const CloseIcon = ({ width = 6, height = 6 }) => (
+import theme from "theme";
+
+const CloseIcon = ({ width = 6, height = 6, color = theme.colors.black }) => (
```

Comment:
This is still being kept around because other files use it as a ref, even though you used MUI icons, correct?